# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1368729174 14400
#      Thu May 16 14:32:54 2013 -0400
# Node ID c7d4146c570d577bfaaec8768eea0fa2e3a505a6
# Parent  0c9aaceadc333e7c6125c441b2d8b9e1d38a1614
check for Qt build tools with $host prefix

* configure.ac: Use AC_CHECK_TOOLS instead of AC_CHECK_PROGS to check
for moc, uic, rcc, and lrelease.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2594,20 +2594,20 @@ if test $build_gui = yes; then
     ## Check for Qt4
     if ! `$PKG_CONFIG --atleast-version=4.0.0 QtCore`; then
       AC_MSG_WARN([Qt >= 4.0.0 not found -- disabling GUI])
       build_gui=no
     fi
   fi
 
   if test $build_gui = yes; then
-    AC_CHECK_PROGS(MOC, [moc-qt5 moc-qt4 moc])
-    AC_CHECK_PROGS(UIC, [uic-qt5 uic-qt4 uic])
-    AC_CHECK_PROGS(RCC, [rcc])
-    AC_CHECK_PROGS(LRELEASE, [lrelease-qt5 lrelease-qt4 lrelease])
+    AC_CHECK_TOOLS(MOC, [moc-qt5 moc-qt4 moc])
+    AC_CHECK_TOOLS(UIC, [uic-qt5 uic-qt4 uic])
+    AC_CHECK_TOOLS(RCC, [rcc])
+    AC_CHECK_TOOLS(LRELEASE, [lrelease-qt5 lrelease-qt4 lrelease])
     if test -n "$MOC" && test -n "$UIC" && test -n "$RCC" && test -n "$LRELEASE"; then
       AC_DEFINE(HAVE_QT, 1, 
         [Define to 1 if Qt is available (libraries, developer header files, utility programs (moc, uic, rcc, and lrelease))])
     else
       AC_MSG_WARN([Qt utility programs moc, uic, rcc, and lrelease not found -- disabling GUI])
       build_gui=no
     fi
   fi
